test/translation/transformation/topLevelVariables.ts

```diff
@@ -0,0 +1,11 @@
+const obj = { value1: 1, value2: 2 };
+const value1 = obj.value1;
+const { value2 } = obj;
+
+let noValueLet;
```

Comment:
I wonder if it would be better to transform it to a nil assignment instead of just dropping it?

Reply:
Maybe we should decide and do this in another PR